src/libsync/configfile.h

```diff
@@ -42,7 +42,7 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     static QString configPath();
     static QString configFile();
     static bool exists();
-
+    const std::chrono::milliseconds DefaultRemotePollInterval { 30000 }; // default remote poll time in milliseconds
```

Comment:
Now there is really no need to make the default public available :D

Also I guess we should use seconds more often instead of milliseconds